tests/integration/test_constraints.py

```diff
@@ -166,11 +166,11 @@ def test_ensure_constraints(memgraph):
         MemgraphConstraintUnique(
             "NodeThree",
             (
-                "attribtue1",
-                "attribtue2",
+                "attribute1",
+                "attribute2",
             ),
         ),
-        MemgraphConstraintUnique("NodeThree", ("attribtue3",)),
+        MemgraphConstraintUnique("NodeThree", ("attribute3",)),
```

Comment:
Sometimes typos are intentional in tests. I saw no evidence of that here.

Reply:
you're probably right